include/http_protocol.h

```diff
@@ -67,6 +67,31 @@ AP_DECLARE(request_rec *) ap_create_request(conn_rec *c);
  */
 request_rec *ap_read_request(conn_rec *c);
 
+/**
+ * Tokenize a HTTP/x.x request line.
+ * @param r the request for which to parse the line
+ * @param line the line to tokenize, will be written to.
```

Comment:
The returned `line` is normalized? garbage?
Maybe a const char *line here could be copied then tokenized?
(for instance it seems that ap_parse_request_line() will overwrite r->the_request)

Reply:
This method was, I think, static before and is now exposed. So, it would make sense to give it a cleaner calling convention at the cost of an additional pstrdup().

Reply:
Actually, it has become purely internal to mod_http1 now. So, I'll remove it from the API and use it internally only.

Reply:
Updated PR with change.